dbschema/seeds/014.product.edgeql

```diff
@@ -1090,20 +1090,15 @@ with
     "project": "South Downs",
     "title": "Recording of Lametations",
     "description": "To help them understand it better.",
-    "mediums": [
-      "Other"
-      ],
+    "mediums": ["Other"],
     "steps": ["Completed"],
     "progressTarget": 100,
     "progressStepMeasurement": "Percent"
   },
   {
     "project": "Misty Mountains",
     "title":  "7 new songs written and recorded",
-    "mediums": [
-      "Audio",
-      "Web"
-      ],
+    "mediums": ["Audio","Web"],
```

Comment:
We want spaces in between our array entries, so like this here...and then on any others.

```suggestion
    "mediums": ["Audio", "Web"],
```